static/js/around-map.js

```diff
@@ -1,3 +1,9 @@
+// init localStorage
+window.onload = _ => {
+localStorage.removeItem('userPosition')
```

Comment:
yes but I can't have access before my componant was mounted so I try many different way ... and with a let that not working ...